tiup/tiup-cluster-topology-reference.md

```diff
@@ -142,10 +142,10 @@ A `server_configs` configuration example is as follows:
 ```yaml
 server_configs:
   tidb:
-    run-ddl: true
     lease: "45s"
     split-table: true
     token-limit: 1000
+    instance.tidb_enable_ddl: true
```

Comment:
It's ON in Line 757 in [tidb-configuration-file.md](https://github.com/pingcap/docs/pull/10910/files#diff-0c6e827c2bde2f920447a23365ce5f4603fe7067d99ec1d1dfc3d22bb64c51e3)

Reply:
It should be `true` according to the `show config` result.